### PR TITLE
Add io.github.qalculate.Qalculate (qalculate-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**Qalculate!** is a multi-purpose cross-platform desktop calculator. It is simple to use but provides power and versatility normally reserved for complicated math packages, as well as useful tools for everyday needs (such as currency conversion and percent calculation). Features include a large library of customizable functions, unit calculations and conversion, symbolic calculations (including integrals and equations), arbitrary precision, uncertainty propagation, interval arithmetic, plotting, and a user-friendly interface (GTK+ and CLI).
https://github.com/scx/flathub/tree/io.github.qalculate.Qalculate
https://github.com/scx/qalculate-flatpak

See also:
http://qalculate.github.io
https://github.com/Qalculate/qalculate-gtk
https://github.com/scx/qalculate-gtk

### CODE REVIEW

**Comment by scx** [comment]
Upstream participation request:
https://github.com/Qalculate/qalculate-gtk/issues/82

**Comment by nedrichards** [comment]
bot, build io.github.qalculate.Qalculate
testbot, build io.github.qalculate.Qalculate

**Comment by scx** [comment]
It looks like a bug on the Flathub side.
https://flathub.org/buildbot/#/builders/3/builds/127

**Comment by scx** [comment]
bot, build io.github.qalculate.Qalculate

**Comment by gusbemacbe** [comment]
You did not merge since February. I need to install it, @nedrichards and @scx . Please do not keep this app under revision for a long time.

**Comment by scx** [comment]
bot, build io.github.qalculate.Qalculate

**Comment by scx** [comment]
@gusbemacbe 

> You did not merge since February. I need to install it, @nedrichards and @scx . Please do not keep this app under revision for a long time.

As you can see, the Flathub maintainers are either too busy or just don't care. Unfortunately, there is nothing I can do about it. I've updated the program and runtime to the latest versions, but that's it.

If you want, you can build the package yourself by using the flatpak-builder tool:
https://github.com/scx/qalculate-flatpak#instructions

**Comment by scx** [comment]
bot, build io.github.qalculate.Qalculate

**Comment by scx** [comment]
I've provided a single-file bundle for x86_64.
https://github.com/scx/qalculate-flatpak/releases

**Comment by scx** [comment]
bot, build io.github.qalculate.Qalculate

**Comment by scx** [comment]
Standalone package for **Qalculate** _3.5.0_:
https://github.com/scx/qalculate-flatpak/releases
https://github.com/scx/qalculate-flatpak/releases/tag/v3.5.0+20191027

File: io.github.qalculate.Qalculate.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+app-id: io.github.qalculate.Qalculate
+runtime: org.gnome.Platform
+runtime-version: '3.32'
```
Comment (LeandroStanger) [inline io.github.qalculate.Qalculate.yaml]: Update runtime to 3.34

**Comment by scx** [comment]
bot, build io.github.qalculate.Qalculate

**Comment by scx** [comment]
Single-file bundle:
https://github.com/scx/qalculate-flatpak/releases
https://github.com/scx/qalculate-flatpak/releases/tag/v3.5.0+20191123

File: io.github.qalculate.Qalculate.yaml (hunk 1 above)
Comment (scx) [reply io.github.qalculate.Qalculate.yaml]: When this application was submitted, the GNOME runtime was up to date. I've updated it to `3.32`, then to `3.34`. I see no point for further updates if this won't be merged soon.

**Comment by scx** [comment]
bot, build io.github.qalculate.Qalculate

**Comment by scx** [comment]
Single-file bundle:
https://github.com/scx/qalculate-flatpak/releases
https://github.com/scx/qalculate-flatpak/releases/tag/v3.6.0+20200101

**Comment by bilelmoussaoui** [comment]
Replaced by #1613

**Pull request closed without merging**